manifests/filter/open_tracing.pp

```diff
@@ -0,0 +1,182 @@
+# == Resource:  repose::filter::open_tracing
+#
+# This is a resource for enabling emitting traces to an OpenTracing tracer (current supports Jaeger).
+# Only usable in >= 8.8.3.0
+#
+# === Parameters
+#
+# [*ensure*]
+# Bool. Ensure config file present/absent
+# Defaults to <tt>present</tt>
+#
+# [*filename*]
+# String. Config filename.
+# Defaults to <tt>open-tracing.cfg.xml</tt>
+#
+# [*service_name*]
+# String. Application name
+# Defaults to <tt>repose</tt>
+#
+# [*connection_type*]
```

Comment:
This seems unnecessary, it can be determined by other configuration, so why include it?

Reply:
Since there are a number of defaults, I didn't see a way of figuring out which transport you want to use.  Do you have an example of how I can figure it out instead of this attribute?